js/links.js

```diff
@@ -9,23 +9,23 @@ window.addEventListener("load", () => {
         h("li", {}, h("a", {href: "chrome://crash"}, "chrome://crash")),
     );
 
-  let btn = h("button", {}, "notify (via info message) with deep-link to internal page");
-    btn.onclick = () => {
+    let btn1 = h("button", {}, "notify (via info message) with deep-link to internal page");
+    btn1.onclick = () => {
         window.webxdc.sendUpdate({ payload: "",  info: "click me to open ./page.html directly", href: "./page.html", notify:["all", window.webxdc.selfAddr]}, "");
     };
-    ul.append(h("li", {}, btn));
+    ul.append(h("li", {}, btn1));
 
-  let btn = h("button", {}, "notify (via summary) with deep-link to internal page");
```

Comment:
indeed that was a copy+paste bug, could have been solved by just removing the `let` from this line